### PR TITLE
MODUSERBL-53 Adding functionality of sending notification

### PR DESCRIPTION
**Pull request by dmytrotur**
MODUSERBL-53 Adding functionality of sending notification to /bl-users/forgotten/password and /bl-users/forgotten/username
This pull request is based on branch MODUSERBL-52 that is not merged to master yet.
To see changes use:  [Comparing changes MODUSERBL-52 MODUSERBL-53](https://github.com/folio-org/mod-users-bl/compare/MODUSERBL-52...MODUSERBL-53)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 4.3.0 2018-11-27
```
Comment (opochepam) [inline NEWS.md]: increment version, this one is already used in https://github.com/folio-org/mod-users-bl/pull/51/files

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "users-bl",
-      "version": "3.2",
+      "version": "3.3",
```
Comment (opochepam) [inline descriptors/ModuleDescriptor-template.json]: the same

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <groupId>org.folio</groupId>
   <artifactId>mod-users-bl</artifactId>
   <name>mod-users-bl</name>
-  <version>4.2.0-SNAPSHOT</version>
+  <version>4.3.0-SNAPSHOT</version>
```
Comment (opochepam) [inline pom.xml]: the same

File: NEWS.md (hunk 1 above)
Comment (dmytrotur) [reply NEWS.md]: Done. Added news for new version.

File: descriptors/ModuleDescriptor-template.json (hunk 2 above)
Comment (dmytrotur) [reply descriptors/ModuleDescriptor-template.json]: The only change in module descriptor is permissions, so i think i should keep version the same as in MODUSERBL-52

File: pom.xml (hunk 3 above)
Comment (dmytrotur) [reply pom.xml]: Done. Updated to version 4.3.1-SNAPSHOT

**Pull request merged**